.github/workflows/release.yaml

```diff
@@ -17,6 +17,8 @@ jobs:
           RELEASE_VERSION="${RELEASE_VERSION:1}"
           ./make_package.sh "$RELEASE_VERSION"
 
+          ./make_package.sh "$RELEASE_VERSION" 4.1-beta1
```

Comment:
Do we need to call `hub release create` a second time?

Reply:
no because the extra `make_package.sh` call will create 2 more bundles along side the existing bundles. `hub release create` is then called with `"${assets[@]}"`, which should be all of the `.tar.gz` and `.zip` files in the packaging directory, including the first agent bundle and the new agent bundle